### ISSUE TITLE
Enhancement: Increase baud to 100MBPS (currently limited to 10MBPS)

### ISSUE THREAD

**Issue by g4jc** [OpenVPN/tap-windows#3]
This issue was brought up before by someone on the mailing list, for people with higher speed I agree this is slowww -
http://openvpn.net/archive/openvpn-users/2004-02/msg00039.html

Please change adapter settings to allow at least 100MBPS.

**Comment by war59312** [comment]
It's now 100 Mbps on Win 8.1 x64. But that still slows down anyone using 1 Gbps.

Why not make it 10 Gbps like LogMeIn Hamachi does.

**Comment by g4jc** [comment]
@war59312 Yes, that's an even better idea. Unfortunately, this project appears inactive. I'm unsure where OpenVPN development is taking place, or if it is still taking place.

**Comment by avi-zs** [comment]
Is it 100 Mbps on windows7?
I am seeing 100 Mbps on Windows 8 but on windows 7, its still showing 10 Mbps.

**Comment by war59312** [comment]
@avi-zs Your right, 100 Mbps seems to be Win 8 and Win 10 only.

@g4jc Yeah, lame. :(

**Comment by 578141611** [comment]
So can we solve this speed limit problem?
